web/public/source/scripts/app.js

```diff
@@ -32,14 +33,21 @@
         'app.services'
     ]).
 
-    run(['$rootScope', 'entries', 'client', function ($rootScope, entries, client) {
-        var defaultUser = {
+    run(['$rootScope', '$cookies', 'entries', 'client', function ($rootScope, $cookies, entries, client) {
+        $rootScope.defaultUser = {
             name: "Guest"
         };
-        $rootScope.defaultUser = defaultUser;
-        $rootScope.selectedUser = defaultUser;
-        entries.getUserEntries(defaultUser);
+        
+        
         client.load();
+        $rootScope.selectedUser = $cookies.getObject('selectedUser');
```

Comment:
The cookie name should probably be set as an constant somewhere. It's not a big issue but since it's used in more than one place it's good practice.